content/applications/services/helpdesk/overview/reports.rst

```diff
@@ -33,26 +34,27 @@ more, click the :guilabel:`Measures` button, and select one or more options from
 menu:
 
 - :guilabel:`Average Hours to Respond`: average number of working hours between a message sent from
-  the customer and the response from the support team. *This is does not include messages sent when
-  the ticket was in a folded stage*
+  the customer and the response from the support team. *This does not include messages sent when
+  the ticket was in a folded stage.*
 - :guilabel:`Hours Open`: number of hours between the date the ticket was created and the closed
   date. If there is no closed date on the ticket, the current date is used. **This measure is not
```

Comment:
```suggestion
  date. If there is no closed date on the ticket, the current date is used. **This measure is not
```
since the previous definition (and other ones after) seem to use *italics* and not **bold**, do you think changing this last sentence to *italics* would be best to make things appear more uniform? Or, do you think the **bold** formatting is necessary for the message that's being delivered. Just something for you to consider during your revisions. Thanks! 👍

Reply:
I do want to keep this different just to make sure there is emphasis on when measures are specific to working hours (and therefore follow the working calendar) and when they aren't, since I think that can be a bit confusing for helpdesk.
If there is a ticket that has been "open" for 24 hours, that is tracking time when the team is off the clock. But if you just want to see how long the team is actively working on a ticket, you would select "Hours to close", because that is only going to show the "working hours", so it would not include off work time. (I hope this clears things up).